src/themes/default/components/core/blocks/Auth/ForgotPass.vue

```diff
@@ -8,7 +8,14 @@
         <form @submit.prevent="sendEmail" novalidate>
           <div class="mb35">
             <p class="mb45">Enter your email to receive instructions on how to reset your password.</p>
-            <input ref="email" class="brdr-none py10 h4 weight-200" type="email" name="email" v-model="email" autocomplete="email" placeholder="E-mail address *">
+            <input
+              ref="email"
+              class="brdr-none brdr-bottom brdr-c-lightgray-secondary py10 h4 weight-200 roboto"
```

Comment:
We have classes for serif i sans-serif font - please use them instead of font-specific classes like `roboto`

Reply:
I found only `.serif` class in `text.scss`, so I added also class for roboto font:
```
.serif { font-family: 'Playfair Display', serif; }
.roboto { font-family: 'Roboto', sans-serif; }
```
I can just rename it to `sans-serif`